src/main/java/org/fundacionjala/pivotal/api/APIProjectsRequests.java

```diff
@@ -0,0 +1,79 @@
+package org.fundacionjala.pivotal.api;
+
+import com.jayway.restassured.RestAssured;
+import com.jayway.restassured.response.Response;
+import java.util.Map;
+import static com.jayway.restassured.RestAssured.given;
+
+/**
+ * APIProjectsRequests.java
+ * Class that implements the methods to get, post, delete and put through Pivotal API.
+ */
+public class APIProjectsRequests {
+
+    private static String projectId = "0";
+    private Response response;
+
+    /**
+     * Constructor.
+     */
+    public APIProjectsRequests() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)